src/app/kvm/components/LXDVMsTable/VMsTable/_index.scss

```diff
@@ -30,5 +30,13 @@
     .pool-col {
       @include breakpoint-widths(0, 0, 0, 0, 12%);
     }
+
+    .select-all-dropdown {
+      display: inline-flex;
+      margin-left: -0.8rem;
+      margin-top: -0.5rem;
+      padding: 0.5rem;
+      margin-right: 0.5rem;
```

Comment:
All of the `0.5rem` values here can be replaced with `$spv--small` or `$sph--small`